internal/store/networkpolicy_test.go

```diff
@@ -0,0 +1,59 @@
+package store
+
+import (
+	"testing"
+
+	networkingv1 "k8s.io/api/networking/v1"
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+	"k8s.io/kube-state-metrics/pkg/metric"
```

Comment:
This should have been caught by our linters. First-party import are to be separated from third-party ones.
```suggestion

	"k8s.io/kube-state-metrics/pkg/metric"
```